Sources/Apodini/Semantic Model Builder/Model/ParameterNamespace.swift

```diff
@@ -46,10 +46,10 @@ public extension Array where Element == ParameterNamespace {
     /// With the `.global` level, a parameter name must be
     /// unique across all `ParameterType`s on the given `Endpoint`.
     /// This is the default namespace when nothing is specified by the exporter.
-    static let global: [ParameterNamespace] = [.all]
+    static let global: [ParameterNamespace] = [.lightweight, .content, .path]
```

Comment:
This seems wrong to me. 
`.global` would require that all parameters are in the same namespace. What you are doing here is `.individual` but without a `.header`.
Replacing `.all` of the old definition would equal to `[[.lightweight, .content, .path, .header]]`.
Meaning an array of namespaces, containing one namespace (the inner array) containing all `ParameterType`s.

I would suggest to leave it as is, especially as with the current version you incorporated, names of `.header` won't ever be checked for collisions.